source/css/sass/mixins/_typography.scss

```diff
@@ -67,6 +67,21 @@
   vertical-align: middle;
 }
 
+@mixin _font-variant-position($position) {
+
+  @if $position == super {
+    top: -0.5em;
+  }
+
+  @supports (font-variant-position: #{$position}) {
+    font-size: inherit;
+    font-variant-position: $position;
+    line-height: normal;
+    position: static;
+    vertical-align: baseline;
```

Comment:
Looks like resetting `line-height` and `vertical-align` doesn't make a difference.

Reply:
Just had a look at the spec, and it says: 'User agents that support the ‘font-variant-position’ property will select a subscript variant glyph and render this without adjusting the baseline or font-size." (see the end of [this section](https://drafts.csswg.org/css-fonts-3/#font-variant-position-prop) )